pennylane/plugins/expt_tensornet.py

```diff
@@ -84,19 +85,25 @@ class TensorNetwork(Device):
         "Identity": identity,
     }
 
+    backend = "numpy"
+    reshape = staticmethod(np.reshape)
+    array = staticmethod(np.array)
+    asarray = staticmethod(np.asarray)
+    real = staticmethod(np.real)
+    imag = staticmethod(np.imag)
+    abs = staticmethod(np.abs)
+
+    C_DTYPE = np.complex128
+    R_DTYPE = np.float64
+
     def __init__(self, wires, shots=1000, analytic=True):
```

Comment:
Implementing the logic for ``analytic`` will be a TODO

Reply:
I've just realised that `analytic=False` doesn't make sense for `tensornet.tf`, as backprop would no longer work. So we should remove this argument from `tensornet.tf`, but add a TODO to get `analytic` working on `tensornet`.